etc/db-config.yaml

```diff
@@ -201,6 +201,14 @@
 -   category: Display
     description: Options related to the DOMjudge user interface.
     items:
+        -   name: default_submission_code_mode
+            type: int
```

Comment:
Let's make this an `array_val`, then one can select which ones are enabled. By default let's enable upload for now.

Reply:
Okay, but in that case, the cookies will also need a major revamp.

Reply:
You should prevent the array from being empty here. How should I handle it when someone submits the form with an empty array?

Should I use an alert?

I haven’t seen any other similar menus to refer to for this implementation.

Sorry about that.

Reply:
I would check that the array doesn't get empty and if it does refuse to save or use a `danger` flash with the message. Alternative is to have a message in the `config-check` page.

Reply:
I think implementing the required restriction in the backend can be done in the next PR, as it's more general. Including it in this commit would make things confusing.

can i do it this way?